--- ./chrome/messenger/content/messenger/am-copies.js	2023-12-07 17:20:26.568665890 +0100
+++ ./chrome/messenger/content/messenger/am-copies.js.new	2023-12-07 17:20:39.944760751 +0100
@@ -486,10 +486,7 @@
  *   archives in
  */
 function updateArchiveHierarchyButton(archiveFolder) {
-  let isGmailImap =
-    archiveFolder.server.type == "imap" &&
-    archiveFolder.server.QueryInterface(Ci.nsIImapIncomingServer).isGMailServer;
-  document.getElementById("archiveHierarchyButton").disabled = isGmailImap;
+    return;
 }
 
 /**
--- modules/MessageArchiver.jsm	2023-12-07 17:19:51.636418186 +0100
+++ modules/MessageArchiver.jsm.new	2023-12-07 17:20:02.380494363 +0100
@@ -253,12 +253,12 @@
 
     let granularity = batch.granularity;
     let forceSingle = !archiveFolder.canCreateSubfolders;
-    if (
-      !forceSingle &&
-      archiveFolder.server instanceof Ci.nsIImapIncomingServer
-    ) {
-      forceSingle = archiveFolder.server.isGMailServer;
-    }
+    // if (
+    //   !forceSingle &&
+    //   archiveFolder.server instanceof Ci.nsIImapIncomingServer
+    // ) {
+    //   forceSingle = archiveFolder.server.isGMailServer;
+    // }
     if (forceSingle) {
       granularity = Ci.nsIMsgIncomingServer.singleArchiveFolder;
     }
